apisix/plugins/ai-aws-content-moderation.lua

```diff
@@ -14,8 +14,12 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 --
+require("resty.aws.config") -- to read env vars before initing aws module
+
 local core = require("apisix.core")
-local aws_instance = require("resty.aws")()
+local aws = require("resty.aws")
+local aws_instance
```

Comment:
Just to confirm:
This basically just represents an http client wrapper right?
I mean, if a user configures this plugin on multiple routes and uses different credentials, they won't interfere with each other?

Reply:
If you can confirm that the following call to `aws_instance:Credentials` does not save in or affect `aws_instance`, we can do so.

Reply:
> I mean, if a user configures this plugin on multiple routes and uses different credentials, they won't interfere with each other?

correct

Reply:
Great!